anime/views.py

```diff
@@ -87,3 +91,22 @@ def anime_delete(id: int):
     db.session.commit()
 
     return '', 204
+
+@animes_bp.route('/api/v1/animes/', methods=['POST'])
+def create_anime():
+    """Route for inserting a new anime resource to DB
+
+    :return: Successful anime input
+    :rtype: AnimeSchema
+    """
+
+    anime_schema = AnimeSchema()
+    try:
+        schema_dict = anime_schema.loads(request.data)
+    except ValidationError as e:
+        return {"error": e.messages}, 400
```

Comment:
Can you use the error handler by raising the exception instead of returning directly 400 here?